client/dashboard/data/domains.ts

```diff
@@ -12,7 +12,7 @@ export enum DomainTypes {
 	TRANSFER = 'transfer',
 }
 
-export interface Domain {
+export interface DomainSummary {
```

Comment:
Do we want to keep this name? The idea is that `DomainSummary` has less information (because it's part of the list) and `Domain` is the full description of the domain.

Right now they have the same format, but I can see this list growing over time.

Reply:
I think it makes sense. Or we do it in a later PR when there's really more things compared to `DomainSummary`.